azure-pipelines/vs-insertion.yml

```diff
@@ -56,6 +56,10 @@ parameters:
     default: 183
     type: number
     displayName: 'Drop Retention Days (do not set to < 90)'
+  - name: SkipUploadingPackages
```

Comment:
Can you elaborate on why you prefer to have this granularity, instead of an input that directly controls `allowPackageConflicts`?

Reply:
I did not see a strong reason for either option. This seemed a tad more clear for the pipeline user and as a bonus it saves like 30s of the pipeline execution.